### DISCUSSION TITLE
why on virtualbox windows startmenu is black, on physical machine blue

### DISCUSSION THREAD

**Discussion by kosuodhmwa** [Open-Shell/Open-Shell-Menu#1562] [Q&A]
### Describe the bug

I've installed a fresh virtualbox vm, then openshell:

![image](https://github.com/Open-Shell/Open-Shell-Menu/assets/24782840/cdc8041c-f8ee-441b-a2f0-6ed5b8a6b1b5)

### Area of issue

Start menu

### To reproduce

Install virtualbox windows with all updates then install most current release OpenShellSetup_4_4_189.exe

### Expected behavior

Blue 
![image](https://github.com/Open-Shell/Open-Shell-Menu/assets/24782840/a4932f99-0a99-42d1-a7f6-18f4cacb689f)

### Open-Shell version

4_4_189

### Windows version

Windows 10 Pro x64 installed on 2023-06-07 (yyyy-mm-dd)

### Additional context

thank you for your feedback

**Comment by ge0rdi** [comment]
Are you sure you have the same Windows and `Open-Shell` settings on both machines?

**Comment by kosuodhmwa** [comment]
yes, win10 pro with mos current updates 

installation timestamp difference a few weeks

**Reply by ge0rdi** [reply]
Looking at your screenshots it seems you are using different skins on your machines.

First one seems to be using `Immersive` skin (introduced just recently).
Second one seems to be using `Metro` skin (was default for long time).

**Comment by kosuodhmwa** [comment]
ok thx-  how to change that? is that an issue by windows or by openshell?

**Reply by ge0rdi** [reply]
You can pick whatever skin you like in `Open-Shell Menu Settings` -> `Skin` tab.

**Comment by kosuodhmwa** [comment]
now i installed one windows 10 more in a physical machine then openshell - same behavoir. no idea what changed on windows with most current version written on installer usb stick?

**Reply by ge0rdi** [reply]
Most likely you've had `Open-Shell` installed on that machine previously. And it simply had different skin selected in settings.
On clean machine (where `Open-Shell` was never installed) you should get `Immersive` skin selected by default (your first screenshot).
